hw/ip/pwm/data/pwm.hjson

```diff
@@ -0,0 +1,201 @@
+            name: "HTBT_EN",
+            desc: '''Modulates blink behavior to create a heartbeat effect. When
+                     HTBT_EN is set, the duty cycle increases (or decreases)
+                     linearly from DUTY_CYCLE.A to DUTY_CYCLE.B and back, in
+                     steps of (BLINK_PARAM.Y+1), with an increment (decrement)
+                     once every (BLINK_PARAM.X+1) PWM cycles. When HTBT_EN is
+                     cleared, the standard blink behavior applies, meaning that
+                     the output duty cycle alternates between DUTY_CYCLE.A for
+                     (BLINK_PARAM.X+1) pulses and DUTY_CYCLE.B for
+                     (BLINK_PARAM.Y+1) pulses.'''
+            resval: 0
+          },
+          { bits: "15:0",
+            name: "PHASE_DELAY",
+            desc: '''Phase delay of the PWM rising edge, in units of 2^(-16) PWM
+                     cycles''',
+            resval: "0x0000"
+          }
+        ]
+      }
+    },
+    { multireg: { name: "DUTY_CYCLE",
+        desc:'''Controls the duty_cycle of each channel.''',
+        count: "NOutputs"
+        swaccess: "rw",
+        cname: "duty_cycle",
+        fields: [
+          { bits: "31:16",
+            name: "B",
+            desc: '''The target duty cycle for PWM output, in units
+                     of 2^(-16)ths of a pulse cycle. The actual precision is
+                     however limited to the (DC_RESN+1) most significant bits.
+                     This setting only applies when blinking, and determines
+                     the target duty cycle.
+                     '''
+            resval: "0x7fff"
+          }
+          { bits: "15:0",
+            name: "A",
+            desc: '''The initial duty cycle for PWM output, in units
+                     of 2^(-16)ths of a pulse cycle. The actual precision is
+                     however limited to the (DC_RESN+1) most significant bits.
+                     This setting applies continuously when not blinking
+                     and determines the initial duty cycle when blinking.
+                     '''
+            resval: "0x7fff"
+          }
+        ]
+      }
+    },
+    { multireg: { name: "BLINK_PARAM",
+        desc: "Hardware controlled blink/heartbeat parameters.",
+        count: "NOutputs"
+        swaccess: "rw",
+        cname: "blink_param",
+        fields: [
+          { bits: "15:0",
+            name: "X",
+            desc: '''This blink-rate timing parameter has two different
+                     interpretations depending on whether or not the heartbeat
+                     feature is enabled. If heartbeat is disabled, a blinking
+                     PWM will pulse at duty cycle A for (X+1) pulses before
+                     switching to duty cycle B. If heartbeat is enabled
+                     the duty-cycle will start at the duty cycle A, but
+                     will be incremented (or decremented) every (X+1) cycles.'''
```

Comment:
Maybe mention that the delta is BLINK_PARAM.Y here? I had to read the two a couple of times to understand what was going on.